src/pages/EnterpriseShared/AppExporter.js

```diff
@@ -160,18 +160,50 @@ export default class AppExporter extends PureComponent {
       );
     }
   };
-  getStatus = status => {
+  getStatus = (status) => {
     if (!status.is_export_before) {
-      return "未导出";
+      return '未导出';
+    }
+    if (status.status == 'success') {
+      return '成功';
     }
-    if (status.status == "success") {
-      return "成功";
+    if (status.status == 'failed') {
+      return '失败';
     }
-    if (status.status == "failed") {
-      return "失败";
+    if (status.status == 'exporting') {
+      return '进行中';
     }
-    if (status.status == "exporting") {
-      return "进行中";
+  };
+  handleVersionInfo = () => {
+    const { exportVersionList, exportVersion } = this.state;
+    const currentVersionInfo = exportVersionList.filter(
+      (item) => item.version === exportVersion
+    );
+    if (currentVersionInfo.length > 0) {
+      this.setState({
+        versionInfo: currentVersionInfo[0]
+      });
+    }
+  };
+
+  handleRelease = (type) => {
+    const { versionInfo } = this.state;
+    const th = this;
+    if (!versionInfo.dev_status) {
```

Comment:
`dev_status` is a string type and should be evaluated correctly.

Reply:
Repair complete